acceleration/fast_training_tutorial.ipynb

```diff
@@ -18,9 +18,9 @@
     "3. Move data to GPU and cache, then execute random transforms on GPU.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #166.                        torch.save(model.state_dict(), "best_metric_model.pt")
</pre><p>I think we should be saving these models to a temporary dir and removing at the end. Otherwise we clutter the local repo with <code>.pt</code> files (even if they are ignored with <code>.gitignore</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/687/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
That's a good point, I updated the PR for it.
Could you please help take a look again?

Thanks.